src/transform/src/lib.rs

```diff
@@ -400,9 +398,7 @@ impl Optimizer {
             Box::new(crate::canonicalize_mfp::CanonicalizeMfp),
             // Identifies common relation subexpressions.
             // Must be followed by let inlining, to keep under control.
```

Comment:
nit: I think you can remove this comment now, as with the proposed setup the lack of explicit inlining entry in the pipeline might cause confusion.

Reply:
Yes!